### ISSUE TITLE
Add log-in with Jacobs University accounts

### ISSUE THREAD

**Issue by kuboschek** [JacobsAlumni/MemberManagement#45]
Allow students to add themselves into the alumni portal sooner,
in turn enabling following features:

* Pre-fill the registration form - less friction during signup for alumni association
* Appearing on the atlas - allowing alumni to find students for collaboration etc..
* Searching the atlas - allowing students to make alumni connections
* Students can sign up for mentorship

Developing these features will also require:

* Updates to the privacy policy / terms
* Data processing / sharing agreement with Jacobs University

The following won't change:
* NO sharing of data between alumni profiles and student profiles unless permission is granted
* NO membership registration for students, faculty or staff via their Jacobs accounts

**Comment by tkw1536** [comment]
I thiink a first step should be "signup" with Jacobs University accounts.

**Comment by kuboschek** [comment]
I agree, that could be the first funtionality. Nonetheless, I want to focus on the long term growth of portal services, and make sure not to limit possibilities due to technical shortcomings.
